src/shop-app.html

```diff
@@ -18,11 +18,25 @@
 <link rel="import" href="../bower_components/iron-media-query/iron-media-query.html">
 <link rel="import" href="../bower_components/iron-pages/iron-pages.html">
 <link rel="import" href="../bower_components/iron-selector/iron-selector.html">
+<link rel="import" href="../bower_components/paper-icon-button/paper-icon-button.html">
```

Comment:
Fixed here.

Hm, for cases where one element uses another one but depends on its container doing a lazy import of it... that's a bit tricky. How do you capture that pattern declaratively? 

The option I went with here was to add a lazy import of `lazy-resources.html` to each of the pages, as they can be sure that those resources will eventually be imported.